tvshows/src/components/features/shows/ShowContainer/ShowContainer.tsx

```diff
@@ -1,101 +1,67 @@
 import { ShowReviewSection } from "@/components/features/review/ShowReviewSection/ShowReviewSection";
 import { fetcher } from "@/fetchers/fetcher";
+import { createReview, deleteReview } from "@/fetchers/mutators";
 import { swrKeys } from "@/fetchers/swrKeys";
-import { IReview, IReviewList } from "@/typings/review.types";
+import { IReview } from "@/typings/review.types";
 import { Container } from "@chakra-ui/react";
 import { useParams } from "next/navigation";
-import { useEffect, useState } from "react";
-import useSWR from "swr";
+import useSWR, { mutate } from "swr";
 import { ShowDetailsCard } from "../ShowDetailsCard/ShowDetailsCard";
 
-const mockReviewList: IReviewList = {
-  reviews: [],
-};
-
 export const ShowContainer = () => {
   const params = useParams();
   const showId = params?.id;
 
   const {
     data: showListResponse,
-    error,
-    isLoading,
+    error: showError,
+    isLoading: showIsLoading,
   } = useSWR(swrKeys.show(Number(showId)), { fetcher });
 
-  const [reviewList, setReviewList] = useState(mockReviewList);
+  const {
+    data: reviewListResponse,
+    error: reviewError,
+    isLoading: reviewIsLoading,
+  } = useSWR(swrKeys.reviews(Number(showId)), { fetcher });
 
-  useEffect(() => {
-    if (showId) {
-      const loadedList = loadFromLocalStorage(Number(showId));
-      setReviewList(loadedList);
+  const addShowReview = async (review: IReview) => {
+    try {
+      await createReview(swrKeys.create_review, { arg: review });
+      console.log(reviewListResponse);
+      mutate(swrKeys.reviews(Number(showId)));
+    } catch (error) {
+      console.error("Error adding review:", error);
     }
-  }, [showId]);
-
-  const saveToLocalStorage = (showId: number, reviewList: IReviewList) => {
-    localStorage.setItem(`reviewlist-${showId}`, JSON.stringify(reviewList));
   };
 
-  const loadFromLocalStorage = (showId: number) => {
-    const reviewListString = localStorage.getItem(`reviewlist-${showId}`);
-    if (!reviewListString) {
-      return mockReviewList;
-    }
+  const deleteShowReview = async (reviewToRemove: IReview) => {
     try {
-      return JSON.parse(reviewListString);
+      await deleteReview(swrKeys.review(Number(reviewToRemove.id)));
+      mutate(swrKeys.reviews(Number(showId)));
```

Comment:
Same comment as for the add review. You can find an example of this in the lecture code.